_docs/setup/cost_tracker.md

```diff
@@ -21,6 +21,8 @@ This is the meter whose tariff will determine the rate the entity consumption is
 ### Entity
 
 This is the entity whose consumption should be tracked and the cost calculated against. This entity should be reporting in `kwh`.
+For example, `sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_accumulative_consumption` for grid import today reported with an Octopus Home Mini,
```

Comment:
suggestion: I'm not a fan of this as it's main purpose is for tracking entities provided outside of the integration, and this example would cause confusion when compared to the cost sensors provided by the integration (as one includes standing charge and one doesn't).

Reply:
I was asked by a user on the GivEnergy customer forum for how to create a HA dashboard sensor that would replicate what they saw on their Octopus account bill.  The cost tracker seemed the best way of doing this to me to create a 'monthly sensor' as they have an OHM and the meter readings would therefore be the same as is used for their bill.

Couldn't do it with a utility meter as there isn't a suitable entity to sum, and would have the need to get the reset point right which the cost tracker does quite easily.

Reply:
I think it's probably best to move this to the FAQ section instead then, as if they come to the cost tracker page, they're probably already thinking about what they want to do. The FAQ entry should also point at https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/849, as if that gets more requests, I'll take a look at it.

Reply:
Yes something in the FAQ's about how to use cost trackers to recreate the Octopus bill would be a good idea.
I personally use the Energy dashboard as you already mention, but the OP wanted a dashboard sensor